### PR TITLE
Accept instance methods bind with queried object

### PR DESCRIPTION
**Pull request by sitleon**
- allow pass instance methods to instance
- allow to convert object to key camel case

### CODE REVIEW

File: lib/qldb.base.model.js [hunk 1]
```diff
@@ -1,4 +1,5 @@
 const { _ } = require('lodash');
+const { toCamelCase } = require('./utils/case-convert');
```
Comment (Steit) [inline lib/qldb.base.model.js]: Question: Why do you want this function in here? What does it add?

Comment (sitleon) [reply lib/qldb.base.model.js]: - this func is using for convert object-key of the passed object to camel-case
- my use-case is I got different naming-standards in js-code-style (camel-case) and database naming (snake-case), ~~I plan to convert it in here~~

File: lib/qldb.base.model.js [hunk 2]
```diff
@@ -132,6 +194,15 @@ class Ledger {
         let args = {}
         args['metadata.id'] = value;
         return await this.qldbConnection.findCommittedData(this.tableName,{where:args});
+
+        return results.map((res) => {
```
Comment (kendrickwkc) [inline lib/qldb.base.model.js:198]: two return

File: lib/qldb.base.model.js [hunk 3]
```diff
@@ -20,27 +21,64 @@ class Ledger {
...
+    }
+
+    /**
+     * build model class instance based on queried object
+     * 
+     * @param {object} raw - queried object
+     * @return {object}
+     */
+    buildModelObject(raw) {
+        let result = raw;
```
Comment (kendrickwkc) [inline lib/qldb.base.model.js:67]: unnecessary assignment?

**Pull request merged**